renku/core/errors.py

```diff
@@ -72,15 +76,30 @@ def return_or_raise(cls, response, expected_status_code):
         raise cls(response)
 
 
+class BadParameter(RenkuException):
```

Comment:
There is a `click.BadParameter` exception as well which might lead to confusion. To enhance readability I would recommend that we come up with a better name? Maybe `InvalidParameter`?

Reply:
I've changed to `ParameterError`.

Reply:
Do we even need our own exception for this? It seems the builtin `ValueError` has pretty much the same semantics.

Reply:
It helps to give a better feedback to user, so, I would prefer to keep it.